mkdocs/utils.py

```diff
@@ -35,6 +36,39 @@
     text_type = unicode         # noqa
 
 
+def yaml_load(source, loader=yaml.Loader):
+    """
+    Custom yaml loader.
+
+    Load all strings as unicode.
+    http://stackoverflow.com/a/2967461/3609487
+    """
+
+    class Loader(loader):
+
+        """Custom Loader."""
+
+        pass
```

Comment:
I think you will find PyLint angry if I remove the pass.  A matter of fact, I think you will find Python angry if I remove the pass and refuse to compile.

Just like you can't have an empty if block without a pass, you can't have an empty class without a pass.

I know it may seem odd, but this is the proper way.  The pass needs to stay.